packages/core/src/types/global.ts

```diff
@@ -0,0 +1,9 @@
+import type { Tagged } from 'type-fest'
+import type { ViewRuleStyle } from './view'
+
+export type GlobalStyleID = Tagged<string, 'GlobalStyleID'>
+
+export interface GlobalStyle {
+  readonly id: GlobalStyleID,
+  readonly styles: ViewRuleStyle[],
+}
```

Comment:
Seems you always have array with single element
Should it be?

```suggestion
export interface GlobalStyle {
  readonly id: GlobalStyleID,
  readonly style: ViewRuleStyle,
}
```

Reply:
For now your suggestion would be better, but I have already prepared these types for style groups coming in next PR. I would prefer to keep it instead of reworking twice if that's ok for you

Reply:
Ah, I see, indeed makes sense.
Maybe make it stricter with `NonEmptyArray`, but this is optional